app/javascript/packs/components/FileOption.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+class FileOption extends React.Component {
+
+  // TODO: handle click of upload button
+  handleUpload() {
+
+  }
+
+  // get link to file already uploaded
+  getFileLink() {
+    if (this.currentFileExists()) {
+      // Need to generate link
+      return "/api/answers/" + this.props.answer.id;
```

Comment:
Do you guys have a routes file on the front end? You should use that and gather all the URLs in one place instead of hard coding it in here

Reply:
We don't have a requester / router at this time. But I agree it's good to have.

@ethanlee16